src/wagtail_ai/static_src/image_description/main.tsx

```diff
@@ -0,0 +1,49 @@
+import './main.css';
+import { fetchResponse } from '../api';
+
+import wandIcon from '!!raw-loader!../wand_icon.svg';
+
+document.addEventListener('wagtail-ai:image-form', (event) => {
+  const input = event.target as HTMLInputElement;
+  const imageId = input.dataset['wagtailai-image-id']!;
+  const csrfToken = (
+    input.form!.querySelector(
+      'input[name=csrfmiddlewaretoken]',
+    ) as HTMLInputElement
+  ).value;
+
+  const flexWrapper = document.createElement('div');
+  flexWrapper.classList.add('wagtailai-input-wrapper');
+  input.parentNode!.replaceChild(flexWrapper, input);
+  flexWrapper.appendChild(input);
+
+  const button = document.createElement('button');
+  button.type = 'button';
+  button.title = input.dataset['wagtailai-button-title'] || '';
+  button.classList.add('wagtailai-button');
+  button.innerHTML = wandIcon;
+  flexWrapper.appendChild(button);
+
+  let errorMessage: HTMLElement | null = null;
```

Comment:
**Suggestion:** We know this will be an `HTMLParagraphElement`, so we might as well define it as such.